hawc/apps/lit/api.py

```diff
@@ -63,7 +63,7 @@ def tagtree(self, request, pk, *args, **kwargs):
                 "Updated (tagtree replace)", assessment, assessment.id, self.request.user.id
             )
         else:
-            raise ValueError()
+            raise ValidationError(f"Request method {self.request.method} is not allowed.")
```

Comment:
The ValueError was being tripped by a HEAD request, which apparently is [treated like a GET request](https://http.dev/head), but won't match any if statements here. Does this seem like the right way to handle it or should we add "HEAD" to the if statement with "GET"?

Reply:
maybe we just add another if clause for head, and return a 200 response w/ no data?